config/templates/managed-velero-operator-csv-template.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: managed-velero-operator-0.0.1
+  namespace: openshift-velero
```

Comment:
I'm not sure if this line is ok, since the other operators have `namespace: placeholder` here. But I wanted to fill it in with a non-placeholder value if possible.

Reply:
The reason value is set to `placeholder` is because this CSV is not directly applied to the cluster, its installed by OLM in the correct namespace when the Catalog operator is notified of a new version through a subscription which is watching the operators catalog source - built and deployed by the app-sre pipleine here (see the `hack/app_sre_create_image_catalog.sh` script).

I'd prefer to keep this as `placeholder` as it matches our other operators and follows the standards in the document here (See CSV Metadata): https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md